apps/wallet-dashboard/lib/utils/migration/groupStardustObjectsByMigrationStatus.ts

```diff
@@ -11,16 +11,16 @@ export type StardustMigrationGroupedObjects = {
 
 export function groupStardustObjectsByMigrationStatus(
     stardustOutputObjects: IotaObjectData[],
-    epochTimestamp: number,
+    epochTimestampMs: number,
     address: string,
 ): StardustMigrationGroupedObjects {
     const migratable: IotaObjectData[] = [];
     const unmigratable: IotaObjectData[] = [];
 
-    const epochUnix = epochTimestamp / 1000;
+    const epochUnix = epochTimestampMs / 1000;
 
     for (const outputObject of stardustOutputObjects) {
-        const outputObjectFields = extractOutputFields(outputObject);
+        const outputObjectFields = extractMigrationOutputFields(outputObject);
 
         if (outputObjectFields.expiration_uc) {
```

Comment:
we should not push an expired output to `unmigratable` array, since that object is not just non-migratable, it just doesn't belong to us. So, I wouldn't have to push to either `unmigratable` or `migratable`, just don't add it anywhere. I think we could fix this in another issue

Reply:
https://github.com/iotaledger/iota/issues/4511